Chap_Terms.tex

```diff
@@ -130,9 +136,8 @@ \section{Naming Conventions}
 \item The \code{pmix_} prefix is used to denote function pointer and type definitions.
 \end{itemize}
 
-Users should not use the \textbf{\code{"PMIX"}}, \textbf{\code{"PMIx"}}, or \textbf{\code{"pmix"}} prefixes in their applications or libraries so as to avoid symbol conflicts with current and later versions of the \ac{PMIx} Standard.
+Users shall not use the \textbf{\code{"PMIX"}}, \textbf{\code{"PMIx"}}, or \textbf{\code{"pmix"}} prefixes in their applications or libraries so as to avoid symbol conflicts with \ac{PMIx} implementations.
 
-%%%%%%%%%%%
```

Comment:
we could fix in the other direction... make this about conflicting with the Standard, but refer to the "naming conflicts" rather than symbol conflicts.